src/generated/graphql/types.ts

```diff
@@ -1575,6 +1575,8 @@ export type Query = {
    * @deprecated Use `getSlateLineup` with a specific SlateLineup instead.
    */
   listTopics: Array<Topic>;
+  /** ----- In Development (2023-08-16) -----Get a slate of ranked recommendations for the MozSocial. */
+  mozSocialSlate: CorpusSlate;
```

Comment:
This is unrelated to this change, and no longer needed. I will create a PR in recommendation-api to remove it.